README.md

```diff
@@ -95,12 +95,41 @@ bun run db:reset
 bun run clean
 ```
 
+## Secrets (local and remote)
+
+We use Bun secrets for local development and SST secrets for remote deployments. Code reads values via `getEnvValue(name)` (see `packages/shared-api/utils/get-env.ts`), which resolves from SST first and falls back to Bun secrets locally.
+
+Secret names:
+
+- LLM:
+  - Bedrock: `BedrockRoleArn`
+  - Vertex: `googleVertexServiceAccountEmail`, `googleVertexAwsProviderAudience`, `GoogleVertexProject`
```

Comment:
should we exclude the prefix `google` because Vertex is the only AI product on GCP? 
And can we keep the naming with capital letters consistent? 

Just out of curiosity, why don't we use constant conversation like `BEDROCK_ROLE_ARN`?